File: Modul 4/Soal 3/PRAK403-2410817210014-ZAVIER GHANI.py
def main():
    angka_pertama, angka_kedua = map(int, input("Masukkan angka pertama dan kedua: ").split())

    if angka_pertama < angka_kedua:
        for i in range(angka_pertama, angka_kedua + 1):
            print(i, angka_kedua - (i - angka_pertama), end=" - " if i != angka_kedua else "")
        print()
    else:
        for i in range(angka_pertama, angka_kedua - 1, -1):
            print(i, angka_kedua + (angka_pertama - i), end=" - " if i != angka_kedua else "")
        print()

main()

File: Modul 4/Soal 3/PRAK403-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    int angka_pertama, angka_kedua;

    printf("Masukkan angka pertama dan kedua: ");
    scanf("%d %d", &angka_pertama, &angka_kedua);

    if (angka_pertama < angka_kedua) {
        for (int i = angka_pertama; i <= angka_kedua; i++) {
            printf("%d %d", i, angka_kedua - (i - angka_pertama));
            if (i != angka_kedua) {
                printf(" - ");
            }
        }
    } 
    else {
        for (int i = angka_pertama; i >= angka_kedua; i--) {
            printf("%d %d", i, angka_kedua + (angka_pertama - i));
            if (i != angka_kedua) {
                printf(" - ");
            }
        }
    }
    printf("\n");

    return 0;
}
